### ISSUE TITLE
question about detector and discriptor 

### ISSUE THREAD

**Issue by ye450450** [Parskatt/DeDoDe#34]
Is there a reason why discriptor and detector used VGG19 network architecture? Not using ResNet or other network architecture

**Comment by Parskatt** [comment]
Not really, you might get some improvement by updating the backbone.

**Comment by Parskatt** [comment]
Vgg19 features tend to be precisely localizable, which is good. See e.g. the ablation in roma. But its usually worse for global matching. You may also want to have a look at more modern backbones not trained on imagenet

**Comment by ye450450** [comment]
and i have one more question. In web-ui code, there is https://github.com/Parskatt/DeDoDe/releases/download/dedode_pretrained_models/dedode_descriptor_B.pth urls for getting the descriptor model. But when i get there, there is no file. I cannot find the file. Can i get the descriptor training recipe? and there is descriptor_L file, too?

**Comment by ye450450** [comment]
Sorry, there is file. But, there is no directory of https://github.com/Parskatt/DeDoDe/releases.

**Comment by Parskatt** [comment]
Correct, those are just files. Code for training models can be found in the experiments folder of the repo.
